spec/models/supplejack_api/user_set_spec.rb

```diff
@@ -220,7 +220,7 @@ module SupplejackApi
     describe "#name" do
       let(:user_set) { FactoryBot.build(:user_set, user_id: User.new.id, url: "1234abc")}
 
-      it "titlizes the name" do
+      it "utitlizes the name" do
```

Comment:
I think this should stay as titlizes the name, meaning that it will convert 'set name' all lower case to 'Set name' uppercasing the first character of the first word in a title to titlize it.

Reply:
You're right!

Reply:
Just fixed it.